src/OpenTelemetry.Instrumentation.Runtime/CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 * Update OTel API version to `1.3.1`.
   ([#631](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/631))
+* Update OTel API version to `1.4.0-beta.1` and change runtime metrics type to ObservableUpDownCounter
+  ([#675](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/675))
```

Comment:
I think that the first entry in the changelog should be removed. The new entry should be split, please consider:

```suggestion
* Update OTel API version to `1.4.0-beta.1`.
  ([#675](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/675))
* Change runtime metrics type to ObservableUpDownCounter
  ([#675](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/675))
```